packages/serialize-request/lib/index.js

```diff
@@ -191,6 +204,16 @@ function createSerializedRequest(properties) {
 	);
 }
 
+/**
+ * @param {any} value
+ *     The value to test.
+ * @returns {value is Iterable<string, string>}
```

Comment:
**suggestion:** I think this needs to change to match the value in the function above to work properly, I got some TS issues when testing and this fixed it:

```suggestion
 * @returns {value is Iterable<[string, string]>}
```